lib/php/test/Integration/Lib/Serializer/JsonSerializeTest.php

```diff
@@ -27,21 +27,13 @@
 
```

Comment:
Looks like we need to change the namespace here:
`Test\Thrift\Unit`
->
`Test\Thrift\Integration\Lib\Serializer`

Reply:
++